src/Microsoft.Dnx.Tooling/Building/BuildManager.cs

```diff
@@ -225,6 +225,7 @@ private bool BuildInternal(string projectPath)
 
                     if (configurationSuccess)
                     {
+                        var packDiagnostics = new List<DiagnosticMessage>();
```

Comment:
Because if the packaging fails, the WriteDiagnostics(allDiagnostics) call is never hit. I thought about refactoring this to be clearer but wanted to limit the change.